### PR TITLE
Add Mantle errors

### PR DESCRIPTION
**Pull request by DylanTinianov**
Ticket: https://smartcontract-it.atlassian.net/browse/BCI-3955
Mantle Checkpoint: https://docs.google.com/document/d/1gD6DovYC51OrlS6n-ciGqeoLDHdED5d5Z_HG3iZ7t8I

### CODE REVIEW

File: core/chains/evm/client/errors.go [hunk 1]
```diff
@@ -250,14 +250,19 @@ var zkEvm = ClientErrors{
 	TerminallyStuck: regexp.MustCompile(`(?:: |^)not enough .* counters to continue the execution$`),
 }
 
+var mantle = ClientErrors{
+	InsufficientEth: regexp.MustCompile(`(: |^)(failed to forward tx to sequencer, please try again. Error message: 'insufficient funds for gas * price + value")`),
+	Fatal:           regexp.MustCompile(`(: |^)failed to forward tx to sequencer, please try again. Error message: 'invalid sender"`),
```
Comment (poopoothegorilla) [inline core/chains/evm/client/errors.go]: dont forget to also add tests

Comment (DylanTinianov) [reply core/chains/evm/client/errors.go]: Will do! I'm not sure if the `insufficient funds for gas * price + value''` is an exact error, or if we're expecting a value in place of the `*`?

File: core/chains/evm/client/errors_test.go [hunk 2]
```diff
@@ -379,6 +380,8 @@ func Test_Eth_Errors_Fatal(t *testing.T) {
 		{"Failed to serialize transaction: max priority fee per gas higher than 2^64-1", true, "zkSync"},
 		{"Failed to serialize transaction: oversized data. max: 1000000; actual: 1000000", true, "zkSync"},
 
+		{"failed to forward tx to sequencer, please try again. Error message: 'invalid sender\"", true, "Mantle"},
```
Comment (Farber98) [inline core/chains/evm/client/errors_test.go]: `'invalid sender\"` has different types of quotes. `'` vs `"` . Is it relevant?

Comment (DylanTinianov) [reply core/chains/evm/client/errors_test.go]: Good catch, just updated for consistency. Either way both will be captured so it doesn't change the expected result.

File: .changeset/seven-kiwis-run.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": patch
+---
+
+Added custom client error messages for Mantle to capture InsufficientEth and Fatal errors. #Added
```
Comment (poopoothegorilla) [inline .changeset/seven-kiwis-run.md]: is changeset case sensitive? why is this failing

Comment (Farber98) [reply .changeset/seven-kiwis-run.md]: it seems the tag expected is `#added` in lowercase

**Review by amit-momin — APPROVED** [review]
LGTM

**Review by huangzhen1997 — APPROVED** [review]

**Review by Farber98 — APPROVED** [review]
LGTM

**Review by prashantkumar1982 — APPROVED** [review]

**Pull request merged**